lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -578,12 +588,26 @@ def main():
         sparsify=dict(default=None, type='bool'),
         openstack_volume_type=dict(default=None),
         image_provider=dict(default=None),
+        host=dict(default=None),
     )
     module = AnsibleModule(
         argument_spec=argument_spec,
         supports_check_mode=True,
     )
 
+    lun = module.params.get('logical_unit')
+    host = module.params['host']
+    # Fail when host is specified with the LUN id. Lun id is needed to identify
+    # an existing disk if already available inthe environment.
+    if host and lun.get("id") is None:
+        module.fail_json(
+            msg="Can not use parameter host ({0!s}) without "
+            "specifying the logical_unit id".format(host)
+        )
+
+    if module._name == 'ovirt_disks':
```

Comment:
Please remove, current devel branch is future 2.8 release, so we don't support plural anymore.

Reply:
Removed